### DISCUSSION TITLE
First Quasar Extension (Quasar Table Crud)

### DISCUSSION THREAD

**Discussion by odranoelBR** [quasarframework/quasar#8904] [Show and Tell]
Hi community!

I exported a component from a project.
Quasar components are awesome, we can break our projects in little extensions and plug in back, injects dependecies ... whit a ready template to do it (https://next.quasar.dev/app-extensions/development-guide/introduction#getting-started)

I think this one can help some projects.
https://github.com/odranoelBR/vue-quasar-crud

**Comment by metalsadman** [comment]
Nice, just a little comment, the sorting seem to be not working.

**Reply by odranoelBR** [reply]
Thanks, @metalsadman !
In the first example, the API doesn't support sorting, the "more examples" the last one does it.

I think I'm going to take this complete API to the getting started.
